models/product-type/src/product-type-draft/transformers.ts

```diff
@@ -0,0 +1,17 @@
+import { Transformer } from '@commercetools-test-data/core';
+import type { TProductTypeDraft, TProductTypeDraftGraphql } from '../types';
+
+const transformers = {
+  default: Transformer<TProductTypeDraft, TProductTypeDraft>('default', {
+    buildFields: ['attributes'],
+  }),
+  rest: Transformer<TProductTypeDraft, TProductTypeDraft>('rest', {
+    buildFields: ['attributes'],
+  }),
+  graphql: Transformer<TProductTypeDraft, TProductTypeDraftGraphql>('graphql', {
+    buildFields: ['attributes'],
```

Comment:
Same here with the field being `attributeDefinitions`
![image](https://user-images.githubusercontent.com/8136478/218781912-3a6a32db-c95e-4558-864e-2262a683d12c.png)

Reply:
🔧  Fixed with [2f4ca81](https://github.com/commercetools/test-data/pull/198/commits/2f4ca8174a0c9595ff0eb70101921a68468f89be)